samtranslator/model/stepfunctions/generators.py

```diff
@@ -129,20 +129,28 @@ def to_cloudformation(self):  # type: ignore[no-untyped-def]
             )
 
         if self.role and self.policies:
-            raise InvalidResourceException(
-                self.logical_id, "Specify either 'Role' or 'Policies' property and not both."
-            )
+            raise InvalidResourceException(self.logical_id, "Specify 'Role' or 'Policies' or neither property.")
```

Comment:
So does it support now both `Role` and `Policy` specified at the same time? I can't see a code change which guarantees it. However the error message is not that clear anymore, that both at the same moment are not allowed.

Reply:
It does not support both Role and Policy at the same time. I am not sure how to word that error message. Saying "Specify 'Role' or 'Policies' or neither property, and not both." Seems wordy.

Reply:
How about "Specify either 'Role' or 'Policies' (but not both at the same time) or neither of them"?